CHANGELOG.md

```diff
@@ -24,3 +24,26 @@ See the contents of the file `python/require.txt` to see the Python package depe
 * moved Github repo to https://github.com/databrickslabs/dbldatagen/releases
 * code tidy up and rename of options
 * added text generation plugin support for python functions and 3rd party libraries
+
+### version 0.3.0-rc1
+
+The code for the Databricks Data Generator has the following dependencies
+
+* Requires Databricks runtime 9.1 LTS or later
+* Requires Spark 3.1.2 or later 
```

Comment:
If we talk only about usage with DBR, why we need to say about Spark 3.1.2?

Reply:
It can be built and used against open source spark. Development and unit tests are run against open source spark

Only license restricts to DBR